crates/sui-node/src/lib.rs

```diff
@@ -615,9 +616,19 @@ impl SuiNode {
         epoch_store: &Arc<AuthorityPerEpochStore>,
         prometheus_registry: &Registry,
     ) -> Arc<ConsensusAdapter> {
+        const REQUEST_TIMEOUT: Duration = Duration::from_secs(30);
```

Comment:
Not sure if 30s is too long for Sui -> worker communications.

Reply:
When we timeout SUI will retry to post the transaction again to worker , but since the changes in the worker which now guarantees that once the transaction is submitted it will eventually get sequenced, it seems to me that a higher timeout would be ok. But again that's my assumption given the current approach.

Reply:
I think higher timeout is OK here.
We did have an issue in the past where narwhal was not starting at all(at least in simtest) and I added this timeout so that issues like that can be surfaced. (otherwise it was just blocking forever on this call)

Reply:
I am surprised there was no timeout here before